README.en.md

```diff
@@ -0,0 +1,143 @@
+[🇧🇷 Português](README.md)
+
+# covid19-br
+
+![pytest@docker](https://github.com/turicas/covid19-br/workflows/pytest@docker/badge.svg)
+
+This repository unifies links and data about reports on the number of cases from the Health State Secretaries (Secretarias Estaduais de Saúde - SES), about the cases of covid19 in Brazil (at each city, daily), amongst other relevant data for analysis, such as deaths by respiratory diseases accounted in the registry (by state, daily).
+
+## License and Quote
+
+The code's license is [LGPL3](https://www.gnu.org/licenses/lgpl-3.0.en.html) and the converted data is [Creative Commons Attribution ShareAlike](https://creativecommons.org/licenses/by-sa/4.0/). In case you use the data, **mention the original font and who treated the data** and in case you share the data, **use the same license**.
+Example of how the data can be quoted:
+- **Source: Secretarias de Saúde das Unidades Federativas, data treated by Álvaro Justen and team of voluntaries [Brasil.IO](https://brasil.io/)**
+- **Brasil.IO: epidemiologic reports of COVID-19 by city daily, available at: https://brasil.io/dataset/covid19/ (last checked in: XX of XX of XXXX, access in XX of XX of XXXX).**
+
+
+## Data
+
+After collected and treated the data stay available in 3 ways on [Brasil.IO](https://brasil.io/):
+
+- [Web Interface](https://brasil.io/dataset/covid19) (made for humanos)
+- [API](https://brasil.io/api/dataset/covid19) (made for humans that develop apps) - [see available API documentation](api.md)
+- [Full dataset download](https://data.brasil.io/dataset/covid19/_meta/list.html)
+
+In case you want to access the data before they are published (ATTENTION: they may not have been checked yet), you can [access directly the sheets in which we are working](https://drive.google.com/open?id=1l3tiwrGEcJEV3gxX0yP-VMRNaE1MLfS2).
+
+If this program and/or the data resulting are useful to you or your company, **consider [donating the project Brasil.IO](https://brasil.io/doe)**, which is kept voluntarily.
+
+
+### FAQ ABOUT DATA
+
+**Before contacting us (we're overloaded) to consult about the data, [CONSULT OUR FAQ](faq.md).**
```

Comment:
```suggestion
**Before contacting us to consult about the data (we're overloaded), [CONSULT OUR FAQ](faq.md).**
```

Reply:
maybe inform the FAQ is "yet?" in Portuguese